product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.2-converting-pox-to-json/1.2.1-pox-to-json-using-payloadfactory-mediator/src/test/java/org/wso2/carbon/esb/scenario/test/PoxToJsonUsingPayloadFactoryTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.esb.scenario.test;
+
+import com.ibm.wsdl.extensions.http.HTTPConstants;
+import org.apache.http.HttpResponse;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioConstants;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioTestBase;
+import org.wso2.carbon.esb.scenario.test.common.http.HTTPUtils;
+import org.wso2.carbon.esb.scenario.test.common.http.HttpConstants;
+import org.wso2.carbon.esb.scenario.test.common.http.RESTClient;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+import static org.wso2.carbon.esb.scenario.test.common.http.HTTPUtils.getResponsePayload;
```

Comment:
We do not need this static import since we haven't used getResponsePayload() function here

Reply:
removed.